akka-docs/src/main/paradox/typed/cluster-singleton.md

```diff
@@ -34,6 +34,73 @@ such as single-point of bottleneck. Single-point of failure is also a relevant c
 but for some cases this feature takes care of that by making sure that another singleton
 instance will eventually be started.
 
+### Singleton manager
```

Comment:
I added this header and the proxy as a work around for the classic doc using the FQCN that is not typed, to work as reference doc for both.